test/boost/tablets_test.cc

```diff
@@ -1487,7 +1487,7 @@ void do_rebalance_tablets(cql_test_env& e,
     auto max_iterations = 1 + get_tablet_count(stm.get()->tablets()) * 10;
 
     for (size_t i = 0; i < max_iterations; ++i) {
-        auto plan = talloc.balance_tablets(stm.get(), load_stats, skiplist, true).get();
+        auto plan = talloc.balance_tablets(stm.get(), load_stats, skiplist).get();
```

Comment:
This patch looks good.
It could be done as part of the test change by the way, and can be committed independently if needed.

Reply:
Yeah, but the test change was independent of tablet options PR which introduced test_mode.